packages/js/app/src/Interfaces/InfoProps.ts

```diff
@@ -0,0 +1,9 @@
+import { IconProp } from '@fortawesome/fontawesome-svg-core';
+
+interface InfoProps {
+    title: string;
```

Comment:
No need for these files, just put them in the component file and export if needed